docs/content/latest/admin/yb-ts-cli.md

```diff
@@ -0,0 +1,229 @@
+
+`yb-ts-cli` is a binary file installed with YugabyteDB and is located in the `bin` directory of the YugabyteDB home directory.
+
+## Syntax
+
+```sh
+yb-ts-cli [ --server_address=<host>:<port> ] <command> <options>
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+- *command*: The operation to be performed. See [Commands](#commands) below.
+- *options*: The options to be applied to the command. See [Options](#options).
+
+## Command line help
+
+To display the available online help, run `yb-ts-cli` without any commands or options (flags) at the YugabyteDB home directory.
+
+```sh
+$ ./bin/yb-ts-cli
+```
+
+## Commands
+
+##### are_tablets_running
+
+If all tablets are running, returns "All tablets are running".
+
+**Syntax**
+
+```sh
+yb-ts-cli [ --server_address=<host>:<port> ] are_tablets_running
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+
+##### count_intents
+
+Print the count of uncommitted intents (or [provisional records](../../../architecture/transactions/ditributed-txns/#provisional-records)). Useful for debugging transactional workloads.
+
+**Syntax**
+
+```sh
+$ ./bin/yb-ts-cli  [ --server_address=<host>:<port> ] count_intents
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+
+##### current_hybrid_time
+
+Prints the value of the current [hybrid time](../../../architecture/transactions/single-row-transactions/#hybrid-time-as-an-mvcc-timestamp).
+
+**Syntax**
+
+```sh
+$ ./bin/yb-ts-cli  [ --server_address=<host>:<port> ] current_hybrid_time
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+
+##### delete_tablet
+
+Deletes the tablet with the specified tablet ID (`tablet_id`) and reason.
+
+**Syntax**
+
+```sh
+$ ./bin/yb-ts-cli  [ --server_address=<host>:<port> ] delete_tablet <tablet_id> "<reason-string>"
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+- *tablet_id*: The identifier (ID) for the tablet.
+- *reason-string*: Text string providing useful information on why the tablet was deleted.
+
+##### dump_tablet
+
+Dump, or export, the specified tablet ID (`tablet_id`).
+
+**Syntax**
+
+```sh
+yb-ts-cli [ --server_address=<host>:<port> ] dump_tablet <tablet_id>
+```
+
+- *host*:*port*: The *host* and *port* of the tablet server. Default is `localhost:9100`.
+- *tablet_id*: The identifier (ID) for the tablet.
+
+##### list_tablets
+
+Lists the tablets on the specified tablet server, displaying the following information:
+
+- Tablet id
+- State
+- Table name
+- Partition
+- Schema
+- properties
+  - lumn_name
```

Comment:
I suspect this should have been Column_name? If not, then maybe this is more confusing than useful, so could remove this?